### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by arrygoo**
Will make changes in the actual labels after this is merged in.

### CODE REVIEW

**Comment by cfnelson** [comment]
@arrygoo made some changes, let me know what you think. If all looks good then lets merge this in 😄 👍 .

File: contributing.md [hunk 1]
```diff
@@ -1,18 +1,65 @@
...
+ * If possible link to example app which replicates the problem.
+ * If it's a security issue, please email us rather than filing an issue.
 
-Pull Requests:
---------------
+###Labelling system:
 
- * Pull requests are always welcome
- * We prioritize PRs over issues
+OK GROW! follows a issue labelling system which is described below.
```
Comment (pauldowman) [inline contributing.md]: _an_ issue _labeling_ system. (_labeling_ with one l is US, 2 l's is British).

File: contributing.md [hunk 2]
```diff
@@ -1,18 +1,65 @@
...
+* **wont-fix**
+  * OK GROW! will not be implementing a fix for the specific issue raised.
+* **duplicate**
+  * Issue has been previously raised, this Issue will be closed and link to the original issue raised.
+* **in-progress**
+  * Issue is currently being worked on to resolve.
+  * *NOTE:* This label is to be removed once completed.
+* **under-consideration**
+  * Enhancement is currently being considered by members of OK GROW!.
+  * *NOTE:* This label is to be removed once the enhancement has been accepted or rejected.
```
Comment (pauldowman) [inline contributing.md:40]: We should add that PRs are still encouraged for _under-consideration_ issues.

File: contributing.md [hunk 3]
```diff
@@ -1,18 +1,65 @@
...
+
+####Additional label:
+Issues labelled with `bug` or `enhancement`, can have one of the following labels applied.
+
+* **will-add**
+  * Enhancement (feature request) will be added to the packages current roadmap.
+* **wont-add**
+  * Enhancement (feature request) will not be added to the packages current roadmap. 
+* **confirmed**
+  * Bug has been confirmed via a reproduction or a member of OK GROW!.
```
Comment (pauldowman) [inline contributing.md]: ...or _by_ a member...

**Comment by pauldowman** [comment]
Nice. Added a few nitpicky grammar comments. :neckbeard:

**Comment by arrygoo** [comment]
@cfnelson & @pauldowman I appreciate the feedback guys! Will make those changes and merge.

File: contributing.md (hunk 2 above)
Comment (cfnelson) [inline contributing.md:40]: @pauldowman Sounds good, I think we should make a point of stating that when a feature-request is under-consideration it doesn't mean that we will accept a PR if one is produced by someone. It has the label under-consideration as we are discussing if we want to support the enhancement and the maintenance cost for adding it. Robert suggested we make that clear to people wanting to contribute to the package.

File: contributing.md (hunk 1 above)
Comment (cfnelson) [inline contributing.md]: 😢

**Pull request merged**